Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -247,6 +247,7 @@ private async Task<bool> ShowJsonTemplatesOperations()
         private async Task<bool> ShowWSDLOperations()
         {
             bool parseSuccess = true;
+            
             AddAPIModelWizard.ProcessStarted();
```

Comment:
please check the changes list to avoid pushing changes like those (new line)  because then it looks like you changed the file in this push while you didn't

Reply:
sure